Cql/Cql.Compiler/TypeManager.cs

```diff
@@ -62,97 +63,127 @@ public TypeManager(TypeResolver resolver, string assemblyName = "Tuples", string
             AssemblyName = assemblyName;
             var assemblyBuilder = AssemblyBuilder.DefineDynamicAssembly(new AssemblyName(AssemblyName), AssemblyBuilderAccess.Run);
             ModuleBuilder = assemblyBuilder.DefineDynamicModule(AssemblyName);
-            Resolver = resolver ?? throw new ArgumentNullException(nameof(resolver));
-
+            Resolver = resolver.ArgNotNull();

```

Comment:
More unnecessary method calls.  The expression you're replacing is the one automatically created by Visual Studio when you Quick Actions => Add Null Check.

Reply:
Reverted